src/cmd-build

```diff
@@ -292,6 +297,34 @@ if [ ! -f /lib/coreos-assembler/.clean ]; then
     src_location="bind mount"
 fi
 
+# And create the ostree repo tarball containing the commit
+ostree_tarfile_path=${name}-${buildid}-ostree.${basearch}.tar
```

Comment:
Let's plan to merge this PR before #794 - In that case I'll add `${basearch}` in #794 and you can leave it off here.